### PR TITLE
Restructure auth docs to be more clear in function

### PR DESCRIPTION
**Pull request by MichaelTrestman**

### CODE REVIEW

File: v21.2/security-reference/authorization.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Authorization, generally, is the control over **who** (users/roles) can perform
 CockroachDB has a unified authorization model, meaning that a given user's permissions are governed by the same policies in different contexts such as accessing the SQL shell or viewing data from the DB Console.
 {{site.data.alerts.end}}
 
+For documentation of procedures related to managing users on CockroachDB clusters, see [Security Reference&mdash;Authorization](security-reference/authorization.html).
```
Comment (MichaelTrestman) [inline v21.2/security-reference/authorization.md:14]:
```suggestion
```

File: v21.2/security-reference/authorization.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ Authorization, generally, is the control over **who** (users/roles) can perform
 CockroachDB has a unified authorization model, meaning that a given user's permissions are governed by the same policies in different contexts such as accessing the SQL shell or viewing data from the DB Console.
 {{site.data.alerts.end}}
 
+
```
Comment (MichaelTrestman) [inline v21.2/security-reference/authorization.md:14]:
```suggestion
```

File: v22.1/authorization.md [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
+title: Managing SQL User Authorization
+summary: Learn procedures for managing the life cycle of SQL users and roles.
 toc: true
 docs_area: manage
 ---
 
+This page documents procedures for managing the life cycle of SQL users and roles on CockroachDB clusters.
+
+For reference documentation and explanation of related concepts, see [Security Reference&mdash;Authorization](security-reference/authorization.html).
+
```
Comment (MichaelTrestman) [inline v22.1/authorization.md:11]:
```suggestion
```

File: cockroachcloud/user-authorization.md [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
-summary: Learn about the user authorization features for CockroachDB Cloud clusters.
+title: Managing User Authorization in CockroachDB Cloud
+summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.
 toc: true
 docs_area: manage
 ---
 
+{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
+
+Before proceding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in full here: [Authorization in {{ site.data.products.db }}](authorization.html)
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:10]:
```suggestion
Before proceeding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in [Authorization in {{ site.data.products.db }}](authorization.html).
```

File: cockroachcloud/user-authorization.md [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
+
+This page documents procedures for managing the life cycle of CockroachDB Cloud users and roles, including:
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:12]:
```suggestion
This page describes the procedures for managing organization users and roles:
```

File: cockroachcloud/user-authorization.md [hunk 6]
```diff
@@ -1,10 +1,21 @@
...
+
+- Adding users to your organization, by creating them or inviting them from another organization
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:14]: Initial caps and periods on list items.

File: cockroachcloud/user-authorization.md [hunk 7]
```diff
@@ -1,10 +1,21 @@
...
+- managing organization-level users roles
+- granting organization users access to clusters through cluster roles
+- managing the privileges associated with cluster roles
+
 {{ site.data.products.db }} requires you to create SQL users to access the cluster.
 
 By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is assigned to the `admin` role. An `admin` SQL user has full privileges for all databases and tables in your cluster. This user can also create additional users and grant them appropriate privileges.
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:21]: Should this info also be in `authorization.md`?

File: cockroachcloud/user-authorization.md [hunk 8]
```diff
@@ -1,10 +1,21 @@
 ---
-title: User Authorization
-summary: Learn about the user authorization features for CockroachDB Cloud clusters.
+title: Managing User Authorization in CockroachDB Cloud
+summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:3]: nit (throughout): I prefer "lifecycle" -- maybe confirm which it should be?

File: cockroachcloud/user-authorization.md (hunk 7 above)
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:21]:
```suggestion
By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
```

File: cockroachcloud/user-authorization.md (hunk 8 above)
Comment (MichaelTrestman) [inline cockroachcloud/user-authorization.md:3]:
```suggestion
summary: Learn procedures for managing the lifecycle of CockroachDB Cloud users and roles.
```

File: cockroachcloud/user-authorization.md (hunk 7 above)
Comment (MichaelTrestman) [reply cockroachcloud/user-authorization.md]: nah, i don't think so, that's more about the model imo. in any case, this is changing with the upcoming release so it'll be moot.

File: cockroachcloud/authorization.md [hunk 9]
```diff
@@ -5,11 +5,11 @@ toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
+{{ site.data.products.db }}'s authorization model has two levels: the {{ site.data.products.db }} organization and clusters within that organization.
 
-In CockroachDB Cloud, an *Organization* corresponds to a an authorization hierarchy rooted in a billing account. All database clusters in {{ site.data.products.db }} belong within an organization. The user of the billing account can create or invite other CockroachDB Cloud users to the organization. 
+A CockroachDB Cloud *organization* corresponds to a an authorization hierarchy rooted in a billing account. Each {{ site.data.products.db }} cluster is created within an organization. The user of the billing account can create or invite other users to the organization, and can grant roles to those users, such as **Cluster Admin** or **Cluster Creator**..
```
Comment (mdlinville) [inline cockroachcloud/authorization.md:10]: Maybe use the Cloud variable.

Maybe something like "The user who created the organization is the owner of the billing account, and can create and invite other users...." WDYT?

File: cockroachcloud/authorization.md [hunk 10]
```diff
@@ -23,20 +23,20 @@ See:
...
-  - Read cluster information in the CockroachDB Cloud Console and Cloud API
+  - Read cluster information using the CockroachDB Cloud Console and Cloud API
 - 'Administrator' allows users to:
   - Create and delete organization users
-  - Create, modify and delete clusters
+  - Create, modify, and delete clusters
   - Create and manage SQL roles on particular clusters
 
-Within a CockroachDB Cloud Organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed over the cluster.
+Within an organization, the unit of database functionality is the *CockroachDB cluster*, which corresponds to a networked set of CockroachDB database nodes. All SQL operations and data storage are distributed across the cluster.
```
Comment (mdlinville) [inline cockroachcloud/authorization.md:32]: This seems like a bit of a more-complicated repeat of the info that is in line 8. WDYT?

File: cockroachcloud/authorization.md [hunk 11]
```diff
@@ -23,20 +23,20 @@ See:
...
 
 Cluster Single Sign-On (SSO) for {{ site.data.products.db }} allows authorized organization users to directly access clusters within the organization via [`ccloud`](ccloud-get-started.html), the {{ site.data.products.db }} command line interface.
 
 However, because organization roles and roles on any given SQL cluster are logically separate, a corresponding SQL role must be created for each SSO organization user, on each particular cluster.
 
-This correspondence lies in the SQL role name, which must be: `sso_{email_name}`, where 'email_name' refers to everything up to the '@'' in an email address. For example, the SQL user `sso_docs` would result from docs@cockroachlabs.com. `ccloud will prompt you to create this user if it does not already exist, in which case an admin must create it manually.
\ No newline at end of file
+This correspondence lies in the SQL role name, which must be: `sso_{email_name}`, where 'email_name' refers to everything up to the '@'' in an email address. For example, the SQL user `sso_docs` would result from docs@cockroachlabs.com. `ccloud` prompts you to create this user if it does not already exist, in which case an admin must create it manually.
```
Comment (mdlinville) [inline cockroachcloud/authorization.md:42]: @nickvigilante Is it a bug to look within code font for spelling errors?

File: cockroachcloud/user-authorization.md [hunk 12]
```diff
@@ -1,24 +1,24 @@
 ---
 title: Managing User Authorization in CockroachDB Cloud
-summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.
+summary: Learn procedures for managing the lifecycle of CockroachDB Cloud users and roles.
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:3]:
```suggestion
summary: Learn how to manage the lifecycle of CockroachDB Cloud organization users and roles.
```

File: cockroachcloud/user-authorization.md [hunk 13]
```diff
@@ -1,24 +1,24 @@
...
 docs_area: manage
 ---
 
 {{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
 
-Before proceding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in full here: [Authorization in {{ site.data.products.db }}](authorization.html)
+Before proceeding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in [Authorization in {{ site.data.products.db }}](authorization.html).
 
-This page documents procedures for managing the life cycle of CockroachDB Cloud users and roles, including:
+This page describes the procedures for managing organization users and roles:
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:12]:
```suggestion
This page shows how to manage {{ site.data.products.db }} organization users and roles:
```

File: cockroachcloud/user-authorization.md [hunk 14]
```diff
@@ -1,24 +1,24 @@
...
-- managing the privileges associated with cluster roles
+- Adding users to your organization, by creating them or inviting them from another organization.
+- Managing organization-level users roles.
+- Granting organization users access to clusters through cluster roles.
+- Managing the privileges associated with cluster roles.
 
 {{ site.data.products.db }} requires you to create SQL users to access the cluster.
 
-By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is assigned to the `admin` role. An `admin` SQL user has full privileges for all databases and tables in your cluster. This user can also create additional users and grant them appropriate privileges.
+By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
```
Comment (mdlinville) [inline cockroachcloud/user-authorization.md:21]:
```suggestion
By default, a new SQL user created by a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
```

File: cockroachcloud/authorization.md (hunk 11 above)
Comment (nickvigilante) [reply cockroachcloud/authorization.md:42]: I believe it is. Can you log it and assign it to me?

Comment (mdlinville) [reply cockroachcloud/authorization.md:42]: https://cockroachlabs.atlassian.net/browse/DOC-7324

File: cockroachcloud/authorization.md (hunk 9 above)
Comment (MichaelTrestman) [reply cockroachcloud/authorization.md:10]: this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.

File: cockroachcloud/authorization.md (hunk 10 above)
Comment (MichaelTrestman) [reply cockroachcloud/authorization.md:32]: this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.

File: cockroachcloud/user-authorization.md [hunk 15]
```diff
@@ -1,10 +1,21 @@
 ---
-title: User Authorization
-summary: Learn about the user authorization features for CockroachDB Cloud clusters.
+title: Managing User Authorization in CockroachDB Cloud
+summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.
 toc: true
 docs_area: manage
 ---
 
+{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
```
[the remark this answers is not in the document]
Comment (MichaelTrestman) [reply cockroachcloud/user-authorization.md:8]: Yup, as noted there, this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.

**Pull request merged**